packages/react/package.json

```diff
@@ -209,6 +208,7 @@
     "yaml": "2.3.4"
   },
   "peerDependencies": {
+    "@primer/primitives": "9.x || 10.x",
```

Comment:
I think a move to this being a peer dependency would be a breaking change on our end (since we would require downstream consumers to now download the package in order to use it)

Could we keep this as a dependency until then? We could add moving it to a peer dependency to our v38 roadmap if that would work

Reply:
CC: @langermank This does sound logical to me, but does this break your strategy for the new release?

Reply:
@joshblack ahh I didn't realize this part of the change is breaking. I think without it we will still be able to unblock dotcom bumps though 👍 

@lukasoppermann I can update the issue to just change the version now and later do a breaking `peerDep` change

Reply:
@joshblack I updated it, can we merge it now?